Octokit/Models/Request/SearchRepositoriesRequest.cs

```diff
@@ -298,37 +298,66 @@ public override string ToString()
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public class DateRange
     {
+        public const string DateTimePattern = "yyyy-MM-dd'T'HH:mm:ss zzz";
+        public const string DatePattern = "yyyy-MM-dd";
+
         private readonly string query = string.Empty;
 
         /// <summary>
         /// Matches repositories with regards to the <param name="date"/>.
         /// We will use the <param name="op"/> to see what operator will be applied to the date qualifier
         /// </summary>
+        [Obsolete("This constructor doesn't use the time component of the specified DateTime. Please use the overload accepting a DateTimeOffset, which also supports time.")]
         public DateRange(DateTime date, SearchQualifierOperator op)
         {
             switch (op)
             {
                 case SearchQualifierOperator.GreaterThan:
-                    query = string.Format(CultureInfo.InvariantCulture, ">{0:yyyy-MM-dd}", date);
+                    query = string.Format(CultureInfo.InvariantCulture, ">{0:DateTimePattern}", date);
                     break;
                 case SearchQualifierOperator.LessThan:
-                    query = string.Format(CultureInfo.InvariantCulture, "<{0:yyyy-MM-dd}", date);
+                    query = string.Format(CultureInfo.InvariantCulture, "<{0:DateTimePattern}", date);
                     break;
                 case SearchQualifierOperator.LessThanOrEqualTo:
-                    query = string.Format(CultureInfo.InvariantCulture, "<={0:yyyy-MM-dd}", date);
+                    query = string.Format(CultureInfo.InvariantCulture, "<={0:DateTimePattern}", date);
                     break;
                 case SearchQualifierOperator.GreaterThanOrEqualTo:
-                    query = string.Format(CultureInfo.InvariantCulture, ">={0:yyyy-MM-dd}", date);
+                    query = string.Format(CultureInfo.InvariantCulture, ">={0:DateTimePattern}", date);
                     break;
             }
         }
 
         /// <summary>
         /// Matches repositories with regards to both the <param name="from"/> and <param name="to"/> dates.
         /// </summary>
-        public DateRange(DateTime from, DateTime to)
+        [Obsolete("This constructor doesn't use the time component of the specified DateTime. Please use the overload accepting a DateTimeOffset, which also supports time.")]
+        public DateRange(DateTime from, DateTime to) 
+        {
+            query = string.Format(CultureInfo.InvariantCulture, "{0:DateTimePattern}..{1:DateTimePattern}", from, to);
+        }
+
+        public DateRange(DateTimeOffset from, DateTimeOffset to)
         {
-            query = string.Format(CultureInfo.InvariantCulture, "{0:yyyy-MM-dd}..{1:yyyy-MM-dd}", from, to);
+            query = string.Format(CultureInfo.InvariantCulture, "{0:DateTimePatten}..{1:DateTimePatten}", from, to);
```

Comment:
This still has the old spelling.  Im not sure you even can parameterize the format string like this anyway?!